src/components/Header/Header.js

```diff
@@ -31,13 +32,35 @@ export default class Header extends React.Component {
           iconClassNameRight='muidocs-icon-navigation-expand-more'
           onLeftIconButtonTouchTap={this.handleToggle}
         />
-        <Drawer open={this.state.open}>
-          <MenuItem
-            onClick={this.handleToggle}
-          >
-            Close Menu
-          </MenuItem>
-          <MenuItem>Menu Item 2</MenuItem>
+        <Drawer
```

Comment:
Maybe this should be extract to menu component?

Reply:
I have to come up with a good solution to split menu and header in its own components. right now it's covered in one which is quite ugly.